pybricksdev/connections/pybricks.py

```diff
@@ -837,10 +837,14 @@ async def write_gatt_char(self, uuid: str, data, response: bool) -> None:
             raise ValueError("Response is required for USB")
 
         self._ep_out.write(bytes([PybricksUsbOutEpMessageType.COMMAND]) + data)
-        # FIXME: This needs to race with hub disconnect, and could also use a
-        # timeout, otherwise it blocks forever. Pyusb doesn't currently seem to
-        # have any disconnect callback.
-        reply = await self._response_queue.get()
+
+        try:
+            reply = await asyncio.wait_for(
+                self.race_disconnect(self._response_queue.get()),
+                timeout=5.0,  # 5-second timeout
```

Comment:
```suggestion
                timeout=5.0,
```

comment is reduandant

Reply:
We could probably make it 1 second. What do you think?

Reply:
Sounds good to me.